Makefile

```diff
@@ -13,7 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-LDFLAGS ?=
+GIT_VERSION ?= $(shell git describe --tags --always --dirty)
+
+LDFLAGS=-buildid= -X github.com/sigstore/gitsign/pkg/version.gitVersion=$(GIT_VERSION)
```

Comment:
I thought the whole point of using `BuildInfo` was so we didn't have to inject this stuff ourselves. Can we get away without it?

Reply:
there is an open issue https://github.com/golang/go/issues/29228 
it is always returning (devel)